lib/RateLimiterStoreAbstract.js

```diff
@@ -192,6 +192,10 @@ module.exports = class RateLimiterStoreAbstract extends RateLimiterAbstract {
    * @returns Promise<RateLimiterRes>
    */
   consume(key, pointsToConsume = 1, options = {}) {
+    if(!Number.isInteger(pointsToConsume)){
```

Comment:
@roggervalf What happens if `pointsToConsume` is string or float? This new check you're adding may be a little destructive for those who already consume points as floats or strings by mistake.

Reply:
thank you @animir and sorry for the delay. I just added few extra validations. I was doing some testing and when an string is passed for example 2.0, redis throws an error because it recognize it as non-integer

Reply:
@roggervalf Does RateLimiterRedis throw error when float number of points is consumed, but it isn't easy to understand the error source? Does your code improve the visibility of the error and doesn't introduce any new error type?

Reply:
@roggervalf Could you fix falling tests?

Reply:
hey @animir sorry for the delay. It should be done

Reply:
Thanks @roggervalf . I had another look at this PR and realized that common `RateLimiterStoreAbstract` maybe not a good place for this new check. What if other limiters work with floats and somebody uses that ability right now? We don't know.

Could you move this change to `_upsert` function in `RateLimiterRedis`? Here https://github.com/animir/node-rate-limiter-flexible/blob/d77383643b20fd3a31d69ede4376c810e6dec29c/lib/RateLimiterRedis.js#L81